reference/7.2/Microsoft.PowerShell.Core/About/about_Redirection.md

```diff
@@ -220,6 +220,52 @@ encoding. If the file has a different encoding, the output might not be
 formatted correctly. To write to files with a different encoding, use the
 `Out-File` cmdlet with its `Encoding` parameter.
 
+### Width of output when writing to a file
+
+When you are writing to a file using either `Out-File` or the redirection
+operators, PowerShell formats table output to the file based on the width of
+the console it is running within. For instance, when logging table output
+to file using a command like e.g. `Get-ChildItem Env:\Path > path.log` on a
+system where the console width is set to 80, the output in the file is
+truncated to 80 characters:
+
+```text
+Name                         Value
+----                         -----
+Path                         C:\Program Files\PowerShell\7-preview;C:\WINDOWS…
+```
+
+Considering that the console width may be set arbitrarily on systems where
+your script is run, you may prefer that PowerShell format table output to
+files based on a width that you specify instead.
+
+The `Out-File` cmdlet provides a `Width` parameter that allows you to set the
+width you would like for table output. Rather than having to add
+`-Width 2000` everywhere you invoke `Out-File`, you can use the
+`$PSDefaultParameterValues` variable to set this value for all usages of the
+`Out-File` cmdlet in a script. And since the redirection operators (`>` and
+`>>`) are effectively aliases for `Out-File`, setting the `Out-File:Width`
+parameter for the whole script impacts the formatting width for the
```

Comment:
Should we make it clear that adding `$PSDefaultParameterValues` in a script makes the change local to the script, but running it from the command line is a global change?

Reply:
I think you have the common case of "user copies script from help example" covered below.  I'll turn that example into a function as you suggest.  Do you think that will be sufficient?

Reply:
Add a link to the about topic.

```
For more information about `$PSDefaultParameterValues`, see [about_Preference_Variables](/powershell/module/microsoft.powershell.core/about/about_preference_variables#psdefaultparametervalues)
```

Reply:
I think the link can be simplified to `about_preference_variables.md#psdefaultparametervalues` based on what I see for the next link in this topic:
```
The `>` operator is not to be confused with the
[Greater-than](about_Comparison_Operators.md#-gt--ge--lt-and--le) comparison
```
If that's not the case let me know and I'll change it.

Reply:
Yep, you are correct. That will work.